examples/projects/typescript-vanilla-with-rollup/package.json

```diff
@@ -33,7 +33,6 @@
     "typescript": "^3.9.7"
   },
   "dependencies": {
-    "bpmn-visualization": "0.11.0",
-    "mxgraph": "4.1.0"
+    "bpmn-visualization": "0.11.0"
```

Comment:
Note: This doesn't work with the 0.11.0, but the next version (0.12.0).

Reply:
yes, so I am putting the PR back to draft 😸

Reply:
Use `0.12.0-0` with 0db33b6

Reply:
All good for me :)